src/Appwrite/Messaging/Adapter/Realtime.php

```diff
@@ -295,8 +295,15 @@ public static function fromPayload(string $event, Document $payload, Document $p
                 $roles = ($collection->getAttribute('permission') === 'collection') ? $collection->getRead() : $payload->getRead();
 
                 break;
-            case strpos($event, 'storage.') === 0:
+            case strpos($event, 'storage.buckets.') === 0:
```

Comment:
reminder to remove this

Reply:
Fixed https://github.com/appwrite/appwrite/pull/2786